frontend/src/queries/nodes/InsightViz/InsightDisplayConfig.tsx

```diff
@@ -58,10 +58,7 @@ export function InsightDisplayConfig(): JSX.Element {
         isLifecycle ||
         ((isTrends || isStickiness) && !(display && NON_TIME_SERIES_DISPLAY_TYPES.includes(display)))
     const showSmoothing =
-        isTrends &&
-        !breakdownFilter?.breakdown_type &&
-        !trendsFilter?.compare &&
-        (!display || display === ChartDisplayType.ActionsLineGraph)
+        isTrends && !breakdownFilter?.breakdown_type && (!display || display === ChartDisplayType.ActionsLineGraph)
```

Comment:
Are we sure rolling averages now properly support "Compare to previous period"? I assume there was an issue previously, but I don't have the context

Reply:
It seems to (from limited testing). @mariusandra do you have any context here?